pcsc-sys/build.rs

```diff
@@ -20,15 +20,22 @@ fn main() {
         }
 
         _ => {
-            pkg_config::Config::new()
-                .atleast_version("1")
-                .probe("libpcsclite")
-                .expect(&format!(
-                    r#"Could not find a PCSC library.
+            let lib_dir = env::var("PCSCLITE_LIB_DIR");
```

Comment:
We can write this directly in the `match` (not using the intermediate `lib_dir` variable) - will be a bit more direct.